app/src/main/java/nerd/tuxmobil/fahrplan/congress/utils/FahrplanMisc.java

```diff
@@ -78,33 +78,22 @@ public static void addAlarm(@NonNull Context context,
         for (String alarmTimeString : alarmTimeStrings) {
             alarmTimes.add(Integer.parseInt(alarmTimeString));
         }
-        long when;
-        Moment moment;
-        long startTime;
-        long startTimeInMilliSec = session.dateUTC;
-
-        if (startTimeInMilliSec > 0) {
-            when = startTimeInMilliSec;
-            startTime = startTimeInMilliSec;
-            moment = Moment.now();
-        } else {
-            moment = session.getStartTimeMoment();
-            startTime = moment.toMilliseconds();
-            when = moment.toMilliseconds();
-        }
```

Comment:
Can you explain why this differentiation has been made? Is there a hint in the Git commit history which you could point to in your commit history to document this change?

Reply:
I don't care for this change. The same differentiation is made inside `Session.getStartTimeMilliseconds()`. So this is not a behavior change.

Having multiple ways to retrieve the start time of a session feels like asking for trouble. This should be fixed when cleaning up the `Session` class. It's probably a larger effort and should be out of scope for this change.

Reply:
Thank you for the hint that this is duplicated within `Session.getStartTimeMilliseconds()` - I haven't seen this.